sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ObjectsShouldNotBeDisposedMoreThanOnce.cs

```diff
@@ -330,70 +167,21 @@ public void Disposed_UsingDeclaration()
                 using var d = new Disposable();
                 d.Dispose(); // Noncompliant {{Refactor this code to make sure 'd' is disposed only once.}}
             }
-
-            public void Implicit_Disposed_UsingDeclaration(Stream str)
-            {
-                using var s = new FileStream("path", FileMode.Open);
-                using var sr = new StreamReader(s); // Noncompliant {{Refactor this code to make sure 's' is disposed only once.}}
-            }
-
-            public void Disposed_Using_Valid()
```

Comment:
this is still useful, right?

Reply:
We have a similar case (not exactly this one) at line 29.

Reply:
I will add back a simplified version of this one